code/core/src/manager/components/sidebar/Search.stories.tsx

```diff
@@ -23,77 +22,101 @@ const getLastViewed = () =>
     .filter((item, i) => item.type === 'component' && item.parent && i % 20 === 0)
     .map((component) => ({ storyId: component.id, refId }));
 
+const setQueryParams = fn().mockName('setQueryParams');
+const getQueryParams = fn().mockName('setQueryParams');
+
 const meta = {
   component: Search,
   title: 'Sidebar/Search',
   parameters: { layout: 'fullscreen' },
+  args: {
+    dataset,
+    getLastViewed: (): Selection[] => [],
+    children: () => <SearchResults {...noResults} />,
+  },
+  render: ({ children, ...args }) => {
+    return <Search {...args}>{children}</Search>;
+  },
```

Comment:
This looks like the default render-function to me, ie. `<Search {...args} />` implicitly has `children` too.

```suggestion
```